src/Microsoft.AspNet.Mvc.Core/FilterActionInvoker.cs

```diff
@@ -219,7 +368,36 @@ private IFilter[] GetFilters()
         private async Task InvokeAllActionFiltersAsync()
         {
             _cursor.SetStage(FilterStage.ActionFilters);
+
+            Debug.Assert(_resourceExecutingContext != null);
+            var bindingContext = new ActionBindingContext()
```

Comment:
So the change here is that `ActionBindingContext` is now owned by `IContextAccessor` and it no longer has properties that refer to services - so model metadata provider and input formatter selector are gone.